lib/v2/showstart/router.js

```diff
@@ -1,5 +1,6 @@
 module.exports = (router) => {
     router.get('/event/:cityCode/:showStyle?', require('./event'));
-    router.get('/search/:keyword?', require('./search'));
-    router.get('/params/:type', require('./params'));
+    router.get('/search/:type/:keyword?', require('./search'));
+    router.get('/artist/:id', require('./artist'));
+    router.get('/brand/:id', require('./brand'));
```

Comment:
Please insert new entries in alphabetical order.
```suggestion
    router.get('/artist/:id', require('./artist'));
    router.get('/brand/:id', require('./brand'));
    router.get('/event/:cityCode/:showStyle?', require('./event'));
    router.get('/search/:type/:keyword?', require('./search'));
```